lib/validator/validator.go

```diff
@@ -301,6 +302,21 @@ func validatePoolMetadata(v *validator.Validate, structLevel *validator.StructLe
 	}
 }
 
```

Comment:
If we are making this change - we should perhaps make the change to disallow IPIP for IPv6 pools.